File: android/app/src/main/java/com/gwangy/nassspandroidsample/HybridEventBridge.kt
package com.gwangy.nassspandroidsample

object HybridEventBridge {
    fun logRequest(message: String) = AdEventLogger.request("hybrid", message)
    fun logLoaded(message: String) = AdEventLogger.loaded("hybrid", message)
    fun logDisplayed(message: String) = AdEventLogger.displayed("hybrid", message)
    fun logClicked(message: String) = AdEventLogger.clicked("hybrid", message)
    fun logFailed(message: String, reason: String) = AdEventLogger.failed("hybrid", message, reason)
}

File: ios/Sources/NapSspIOSSample/HybridEventBridge.swift
import Foundation

enum HybridEventBridge {
    static func logRequest(_ message: String) { AdEventLogger.request(format: "hybrid", id: message) }
    static func logLoaded(_ message: String) { AdEventLogger.loaded(format: "hybrid", id: message) }
    static func logDisplayed(_ message: String) { AdEventLogger.displayed(format: "hybrid", id: message) }
    static func logClicked(_ message: String) { AdEventLogger.clicked(format: "hybrid", id: message) }
    static func logFailed(_ message: String, reason: String) { AdEventLogger.failed(format: "hybrid", id: message, reason: reason) }
}
